### DISCUSSION TITLE
2 finger scroll

### DISCUSSION THREAD

**Discussion by fusionconsulting** [fleaflet/flutter_map#1983] [Q&A]
Just wondering if there's any way to change the scroll behaviour to require 2 fingers rather than 1. Would that be possible with the map layer or would I need to implement a custom gesture recogniser?

**Comment by josxha** [comment]
Try `InteractiveFlag.all & ~InteractiveFlag.drag`. There is another flag [pinchMove](https://pub.dev/documentation/flutter_map/latest/flutter_map/InteractiveFlag/pinchMove-constant.html) that keeps 2 finger movements enabled.

- https://docs.fleaflet.dev/usage/options/interaction-options
- https://pub.dev/documentation/flutter_map/latest/flutter_map/InteractiveFlag-class.html
